core/trino-server-rpm/src/test/java/io/trino/server/rpm/ServerIT.java

```diff
@@ -76,6 +76,9 @@ public void testInstallUninstall()
         // Release names as in the https://api.adoptium.net/q/swagger-ui/#/Release%20Info/getReleaseNames
         testInstall("jdk-21.0.2+13", "/usr/lib/jvm/temurin-21", "21");
         testUninstall("jdk-21.0.2+13", "/usr/lib/jvm/temurin-21", "21");
+
+        testInstall("jdk-22+36", "/usr/lib/jvm/temurin-22", "22");
+        testUninstall("jdk-22+36", "/usr/lib/jvm/temurin-22", "22");
```

Comment:
The `expectedJavaVersion` parameter in `testUninstall` is not used, and Error Prone fails the build for me locally on this. Why isn't it failing on CI?

Reply:
because it skips some mdules - https://github.com/trinodb/trino/blob/0714cae31d3e811810db3d5e4ca3c0e1ea09a943/.github/workflows/ci.yml#L199-L200

Reply:
Is it on purpose?

Reply:
Let's fix both issues

Reply:
not sure why though. It's been there since beginning - https://github.com/trinodb/trino/commit/7949e4125f024103cfdc3b9956141d57e482494a

Reply:
Maybe we thought that there's no Java code in that module (like it's the case for docs and rpm modules)?

Reply:
Oh, sorry, I now realized that *this* is the RPM module :D

Reply:
But yeah, I too would assume that there's no Java code here